frontend/src/components/pages/RegistrantExperience/ReviewRegistration/index.tsx

```diff
@@ -1,62 +1,26 @@
 import React from "react";
 import { Box } from "@chakra-ui/react";
 import PaymentSummary from "./PaymentSummary";
-import { CartItem } from "../../../../types/RegistrationTypes";
 import ReviewInformation from "./ReviewInformation";
+import { CreateCamperRequest } from "../../../../types/CamperTypes";
+import { CampResponse } from "../../../../types/CampsTypes";
+import { mapToCampItems } from "../../../../utils/RegistrationUtils";
 
 type ReviewRegistrationProps = {
-  isChecked: boolean;
-  toggleChecked: () => void;
-  campName: string;
+  campers: CreateCamperRequest[];
+  camp: CampResponse;
 };
 
-const items: CartItem[] = [
-  {
-    name: "yes",
-    campers: 2,
-    totalPrice: 15.5,
-    details: "Some sample details",
-  },
-  {
-    name: "yes",
-    campers: 2,
-    totalPrice: 15.5,
-  },
-  {
-    name: "yes",
-    campers: 2,
-    totalPrice: 15.5,
-  },
-  {
-    name: "yes",
-    campers: 2,
-    totalPrice: 15.5,
-  },
-  {
-    name: "yes",
-    campers: 2,
-    totalPrice: 15.5,
-  },
-  {
-    name: "yes",
-    campers: 2,
-    totalPrice: 15.5,
-  },
-  {
-    name: "yes",
-    campers: 2,
-    totalPrice: 15.5,
-  },
-];
-
 const ReviewRegistration = ({
-  isChecked,
-  toggleChecked,
-  campName,
+  campers,
+  camp,
 }: ReviewRegistrationProps): React.ReactElement => {
   return (
     <Box>
-      <PaymentSummary campName={campName} items={items} />
+      <PaymentSummary
+        campName={camp.name}
+        items={mapToCampItems(camp, campers)}
```

Comment:
is it worth it to just map the camp to cart items once and pass around an array of cart items instead? seems like we are making a lot of calls to this function which could just be replaced by the state

Reply:
Hmm maybe, but it feels like we're storing the same information redundantly though?